apps/ejabberd/src/cyrsasl.erl

```diff
@@ -36,15 +36,12 @@
 
 -include("ejabberd.hrl").
 
--record(sasl_mechanism, {mechanism :: mechanism(),
+-record(sasl_mechanism, {mechanism :: mechanism() | atom(), % atom() used in ets:select
```

Comment:
Can any atom be used in the ets:select or only some well defined like '_' f.e. ?